mne/io/utils.py

```diff
@@ -161,7 +160,11 @@ def _read_segments_file(raw, data, idx, fi, start, stop, cals, mult,
         for sample_start in np.arange(0, data_left, block_size) // n_channels:
             count = min(block_size, data_left - sample_start * n_channels)
             block = np.fromfile(fid, dtype, count)
-            block = block.reshape(n_channels, -1, order='F')
+            try:
+                order = raw._order
+            except AttributeError:
+                order = 'F'
```

Comment:
let's just set `raw._order = 'F' ` in `_BaseRaw`, then we don't need any try/except